src/frontend/src/lib/components/send/SendTokensList.svelte

```diff
@@ -21,18 +21,22 @@
 	$: loading = $erc20UserTokensNotInitialized;
 </script>
 
-<TokensSkeletons {loading}>
-	{#each tokens as token (token.id)}
-		<TokenCardWithOnClick on:click={() => dispatch('icSendToken', token)}>
-			<TokenCardContent {token} />
-		</TokenCardWithOnClick>
-	{/each}
+<div class="stretch">
+	<TokensSkeletons {loading}>
+		{#each tokens as token (token.id)}
+			<TokenCardWithOnClick on:click={() => dispatch('icSendToken', token)}>
+				<TokenCardContent {token} />
+			</TokenCardWithOnClick>
+		{/each}
 
-	{#if tokens.length === 0}
-		<p class="mt-4 mb-6 text-dark opacity-50">{$i18n.tokens.manage.text.all_tokens_zero_balance}</p>
-	{/if}
-</TokensSkeletons>
+		{#if tokens.length === 0}
+			<p class="mt-4 mb-6 text-dark opacity-50">
+				{$i18n.tokens.manage.text.all_tokens_zero_balance}
+			</p>
+		{/if}
+	</TokensSkeletons>
+</div>
 
-<button class="secondary full center text-center" on:click={modalStore.close}>
+<button class="secondary block flex-1 mb-2" on:click={modalStore.close}>
```

Comment:
Now that `ButtonGroup` has been removed, you ned to change the style class?

`block flex-1 mb-2` is needed?

Reply:
good catch! I forgot to revert to the original classes